test/e2e/config/docker.yaml

```diff
@@ -205,10 +205,10 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.24.0"
-  KUBERNETES_VERSION: "v1.24.0"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.23.6"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.24.0"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.25.0"
+  KUBERNETES_VERSION: "v1.25.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.24.0"
```

Comment:
Q: should this be v1.24.4 (all around the places)

Reply:
Not sure if there is another place then here